prowler/lib/scan/scan.py

```diff
@@ -29,11 +30,31 @@ def __init__(self, provider: Provider, checks_to_execute: list[str]):
         """
         self._provider = provider
         # Remove duplicated checks and sort them
-        self._checks_to_execute = sorted(list(set(checks_to_execute)))
+        self._checks_to_execute = (
```

Comment:
Why are you doing this here? I mean the `check[0]`.

Reply:
In this case, `recover_checks_from_provider` returns a tuple with the check name and the check path, as I'm just storing the check name I take the first value from the tuple.

Reply:
Good catch, I forgot about it!